TombEngine/Game/Hud/PickupSummary.h

```diff
@@ -25,8 +25,21 @@ namespace TEN::Hud
 		float StringScale  = 0.0f;
 		float StringScalar = 0.0f;
 
+		Vector2		PrevPosition	= Vector2::Zero;
+		EulerAngles PrevOrientation = EulerAngles::Identity;
+		float		PrevScale		= 0.0f;
+		float		PrevOpacity		= 0.0f;
+
 		bool IsOffscreen() const;
 		void Update(bool isHead);
+
+		void StoreInterpolationData()
```

Comment:
Regarding this, and further additions of `StoreInterpolationData`: this is a case for an abstract class. We should think about inheriting all structs/classes which implement `StoreInterpolationData` from the same base.

Reply:
Agreed